Stack.py

```diff
@@ -0,0 +1,45 @@
+class Stack():
+
+  def __init__(self):
+    pass
+
+  def is_stack_empty(self):
+    pass
+
+  def push(self, x):
+    pass
+
+  def pop(self):
+    pass 
+
+
+def test_stack():
+  #Create stack.
+  s = Stack()
+  
+  #Check if Stack is succesfuly created.
+  assert s != None, f'''The instance is not created successfully.'''
+
+  #Ckeck if s is empty.
+  assert s.is_stack_empty(), f'''At this point stack must be empty.'''
+
+  s_push_item_1 = s.push(x=4)
```

Comment:
The function `push` should not return any value, therefore this variable is useless.